data_upload/extract.py

```diff
@@ -0,0 +1,120 @@
+import os
+import csv
+from datetime import datetime
+from dotenv import load_dotenv
+import boto3
+import psycopg2
+import psycopg2.extras
+from psycopg2.extensions import connection, cursor
+from typing import Any
+
+
+QUERY = """ SELECT
+    e.earthquake_id,
+    e.time,
+    e.magnitude,
+    e.tsunami,
+    e.felt_report_count,
+    e.cdi,
+    e.latitude,
+    e.longitude,
+    e.depth,
+    e.detail_url,
+    a.alert_type,
+    m.magnitude_type,
+    n.network_name,
+    t.type_name
+    FROM earthquakes AS e
+    JOIN alerts AS a ON e.alert_id = a.alert_id
+    JOIN networks AS n ON e.network_id = n.network_id
+    JOIN magnitude AS m ON e.magnitude_id = m.magnitude_id
+    JOIN type AS t ON e.type_id = t.type_id;
```

Comment:
Could you just do? :
```suggestion
    SELECT *
    FROM earthquakes AS e
    JOIN alerts AS a ON e.alert_id = a.alert_id
    JOIN networks AS n ON e.network_id = n.network_id
    JOIN magnitude AS m ON e.magnitude_id = m.magnitude_id
    JOIN type AS t ON e.type_id = t.type_id;
```

Reply:
I did think this, but wouldn't that also select the foreign keys that are in the earthquakes table (magnitude_id, alert_id, etc.)? do we want those ids in the csv, or just the names that those ids correspond to in their own tables?

Reply:
Thats why I was thinking to possibly use dataframes and then reduce the number of columns by doing df = df[COLUMNS_WE_NEED]

Reply:
Okay makes sense, will implement those changes now :)